src/makeRobot.js

```diff
@@ -38,7 +38,80 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const template = {
+    x: 0,
+    y: 0,
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
```suggestion
    name,
    wheels,
    version,
```
Назви сходяться. Можна виокристовувати скорочений запис